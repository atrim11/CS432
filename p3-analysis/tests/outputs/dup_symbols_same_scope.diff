--- outputs/dup_symbols_same_scope.txt	2024-10-24 16:11:20.220640037 -0400
+++ expected/dup_symbols_same_scope.txt	2024-10-24 16:02:43.856587362 -0400
@@ -0,0 +1 @@
+Variable 'x' redeclared on line 5
