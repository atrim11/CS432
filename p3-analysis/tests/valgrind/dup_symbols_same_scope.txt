==2601311== Memcheck, a memory error detector
==2601311== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==2601311== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==2601311== Command: ../decaf inputs/duplicate_symbols.decaf
==2601311== 
Variable 'x' redeclared on line 3
==2601311== 
==2601311== HEAP SUMMARY:
==2601311==     in use at exit: 0 bytes in 0 blocks
==2601311==   total heap usage: 1,762 allocs, 1,762 frees, 203,596 bytes allocated
==2601311== 
==2601311== All heap blocks were freed -- no leaks are possible
==2601311== 
==2601311== For lists of detected and suppressed errors, rerun with: -s
==2601311== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
