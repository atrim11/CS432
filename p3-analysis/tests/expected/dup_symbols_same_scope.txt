Variable 'x' redeclared on line 3
